src/Site.hs

```diff
@@ -4,15 +4,21 @@ module Site where
 import Snap
 import Snap.Snaplet.Heist
 import Snap.Snaplet.PostgresqlSimple
+import Data.ByteString
 
 import Application
+import User.Handler
 
 appInit :: SnapletInit App App
 appInit = makeSnaplet "snap" "SNAP Framework example" Nothing $ do
   hs <- nestSnaplet "heist" heist $ heistInit "templates"
   p <- nestSnaplet "db" db pgsInit
-  addRoutes [("/", welcomePage)]
+  addRoutes routes
   return $ App hs p
 
-welcomePage :: Handler App App ()
+routes :: [(ByteString,AppHandler)]
```

Comment:
@kinorius123 What about moving this into another module? Perhaps: `src/Routes.hs`.